config.go

```diff
@@ -172,6 +173,20 @@ func (h hypervisor) defaultMemSz() uint32 {
 	return h.DefaultMemSz
 }
 
+func (h hypervisor) defaultBridges() uint32 {
+	if h.DefaultBridges <= 0 {
```

Comment:
- This should be `if h.DefaultBridges == 0` since the var is an unsigned.
- Can you add a test for this function?

Reply:
yes, Thanks @jodh-intel